Google.PowerShell.IntegrationTests/Storage/Storage.GcsBucket.Tests.ps1

```diff
@@ -23,6 +23,39 @@ Describe "Get-GcsBucket" {
         gsutil rb gs://gcps-testbucket 2>$null
     }
 
+    It "should take into account changes in the Cloud SDK, including environment variables" {
+        # This key contains service account information that has access to project quoct-test-project
+        $additionalKey = Resolve-Path "$PSScriptRoot\..\AdditionalServiceAccountCredentials.json"
+        if ($null -eq $additionalKey -or (-not (Test-Path $additionalKey))) {
+            Write-Host "Key for additional service account cannot be found. Test is skipped."
```

Comment:
The key is only available in AppVeyor because it has to be decrypted (we store the secret key in AppVeyor).